prime-router/settings/STLTs/NH/nh-dphs.yml

```diff
@@ -0,0 +1,101 @@
+- name: "nh-dphs"
+  description: "New Hampshire Division of Public Health Services"
+  jurisdiction: "STATE"
+  stateCode: "NH"
+  countyName: null
+  filters: null
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "nh-dphs"
+      topic: "full-elr"
+      customerStatus: "active"
+      translation:
+        schemaName: "azure:/metadata/hl7_mapping/receivers/STLTs/NH/NH-receiver-transform.yml"
```

Comment:
Please remove "metadata".  It doesn't work in staging.  It should be "azure:/hl7_mapping ....

Reply:
- same comment as on other PR's either use classpath: or follow Ott's suggestion and add the schema to azure

Reply:
@GilmoreA6 We add the schemas to Azure in the validate tickets